.env.example

```diff
@@ -10,11 +9,7 @@ OAUTH_CLIENT_ID=faf-website
 OAUTH_CLIENT_SECRET=banana
 SESSION_SECRET_KEY=banana
 RECAPTCHA_SITE_KEY=6Lc4GmgaAAAAAPaFbnwuP_dBGG--SSy2Hi5UcqwK
-
-
-TOKEN_LIFESPAN=43200                       # an api token is valid for 12h
-CLAN_INVITES_LIFESPAN_DAYS = 30            # 30 days
-EXTRACTOR_INTERVAL= 5 # 5 minutes
-PLAYER_COUNT_INTERVAL= 15           # 15 seconds
-
-
+TOKEN_LIFESPAN=43200
+CLAN_INVITES_LIFESPAN_DAYS=30
+EXTRACTOR_INTERVAL=5
+PLAYER_COUNT_INTERVAL=15
```

Comment:
Why were the comments removed, in this case at least they help inform about units where the names are insufficient for that

Reply:
was removing the spaces (spaces are not save in env files), but then i thought the comments where kinda useless - at least for me. i can add them again if you liked them

Reply:
I would keep the comments since they indicate the units

Reply:
added them again.